torch/csrc/lazy/ts_backend/ts_node_lowering.cpp

```diff
@@ -158,192 +158,6 @@ torch::lazy::TSOpVector Scalar::Lower(
-  // it will never be aliased, and therefore it's safe.
-  torch::jit::Value* destination =
-      GenerateClone(loctx->GetOutputOp(operand(0)), function);
-
-  // Replay the diagonal.
-  std::vector<torch::jit::NamedValue> arguments;
-  arguments.emplace_back(destination);
-  arguments.emplace_back(offset);
-  arguments.emplace_back(dim1);
-  arguments.emplace_back(dim2);
-  auto diag = LowerBuiltin(at::aten::diagonal, function, arguments);
-
-  // Update the replayed diagonal view with the input.
-  GenerateCopy(diag.front(), loctx->GetOutputOp(operand(1)), function);
-
-  // Destination's diag view should be updated.
-  return {destination};
-}
-
-torch::lazy::TSOpVector Narrow::Lower(
-    std::shared_ptr<torch::jit::GraphFunction> function,
-    torch::lazy::TSLoweringContext* loctx) const {
-  const torch::lazy::Output& input = operand(0);
-  torch::jit::Value* base = loctx->GetOutputOp(input);
-  const torch::lazy::Shape& input_shape = input.shape();
-  TORCH_CHECK_EQ(sizes.size(), base_indices.size());
-  TORCH_CHECK_EQ(input_shape.dim(), base_indices.size());
-  for (size_t dim = 0; dim < base_indices.size(); ++dim) {
-    int64_t start = base_indices[dim];
-    base = GenerateSlice(
-        /*base=*/base,
-        /*dim=*/dim,
-        /*start=*/start,
-        /*end=*/start + sizes[dim],
-        /*step=*/1,
-        /*function=*/function);
-  }
-  return {base};
-}
-
-torch::lazy::TSOpVector NarrowViewUpdate::Lower(
-    std::shared_ptr<torch::jit::GraphFunction> function,
-    torch::lazy::TSLoweringContext* loctx) const {
-  torch::jit::Value* dest =
-      GenerateClone(loctx->GetOutputOp(operand(0)), function);
-  const torch::lazy::Output& source_argument = operand(1);
-  const torch::lazy::Shape& source_shape = source_argument.shape();
-  TORCH_CHECK_EQ(source_shape.dim(), base_indices.size());
-  torch::jit::Value* base = dest;
-  for (size_t dim = 0; dim < base_indices.size(); ++dim) {
-    int64_t start = base_indices[dim];
-    base = GenerateSlice(
-        /*base=*/base,
-        /*dim=*/dim,
-        /*start=*/start,
-        /*end=*/start + source_shape.size(dim),
-        /*step=*/1,
-        /*function=*/function);
-  }
-  GenerateCopy(base, loctx->GetOutputOp(source_argument), function);
-  return {dest};
-}
-
-torch::lazy::TSOpVector Permute::Lower(
-    std::shared_ptr<torch::jit::GraphFunction> function,
-    torch::lazy::TSLoweringContext* loctx) const {
-  std::vector<torch::jit::NamedValue> arguments;
-  arguments.emplace_back(loctx->GetOutputOp(operand(0)));
-  arguments.emplace_back(dims);
-  return LowerBuiltin(this, function, arguments);
-}
-
-torch::lazy::TSOpVector Resize::Lower(
-    std::shared_ptr<torch::jit::GraphFunction> function,
-    torch::lazy::TSLoweringContext* loctx) const {
-  std::vector<torch::jit::NamedValue> arguments;
-  for (const torch::lazy::Output& output : operands()) {
-    arguments.emplace_back(loctx->GetOutputOp(output));
-  }
-  return LowerBuiltin(this, function, arguments);
```

Comment:
what's this LowerBuiltin? Should we remove this function too?

Reply:
It's defined above and still in use. So, we shouldn't remove it